src/command.py

```diff
@@ -29,16 +29,15 @@ def cursorless_single_target_command(
             *args,
         )
 
-    def cursorless_single_target_command_args(
+    def cursorless_single_target_command_with_arg_list(
```

Comment:
Ok sorry to keep dropping nits on this one, but now that I think of it, do we want to do the reverse, and implement `cursorless_single_target_command` in terms of this action?  Seems cleaner?

Reply:
Year that might be better we probably can talk about it

Reply:
Would this be worth doing before merge?

Reply:
Is not crucial so we can do it before or later. Go ahead and do the march if you want.